.gitignore

```diff
@@ -59,3 +59,5 @@ specialexport.json
 
 # Custom CSS file
 web/custom.css
+
+yarn-error.log
```

Comment:
to match the Webpack Encore Bundle recipe, see https://github.com/symfony/recipes/blob/main/symfony/webpack-encore-bundle/1.10/manifest.json#L16